crates/bevy_pbr/src/material.rs

```diff
@@ -9,6 +10,8 @@ pub struct StandardMaterial {
     pub albedo: Color,
     #[shader_def]
     pub albedo_texture: Option<Handle<Texture>>,
```

Comment:
I propose to rename albedo to base_color_factor and albedo_texture to base_color_texture, because it actually doubles as the specular for metallic materials and the diffuse for non-metallic materials.

Reply:
Done.